src/ResourceManager/Profile/Commands.Profile/help/Send-Feedback.md

```diff
@@ -0,0 +1,36 @@
+---
```

Comment:
This did not make any change to: https://github.com/Azure/azure-powershell/blob/preview/src/ResourceManager/Profile/Commands.Profile/help/AzureRM.Profile.md, if you geenrate help for the module using platyPS, this should be taken care of